Directory.Build.props

```diff
@@ -4,9 +4,8 @@
     <!-- Include PDBs in Nuget package. -->
     <AllowedOutputExtensionsInPackageBuildOutputFolder>.pdb; $(AllowedOutputExtensionsInPackageBuildOutputFolder)</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <!-- Copy all project dependencies to bin folder -->
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
@DavideGCosta it turned out to be this line that was causing the problems. It was not easy to find the issue....but we did.